packages/npm-packages/ruby-wasm-wasi/test-e2e/integrations/browser-script.spec.ts

```diff
@@ -60,4 +60,85 @@ if (!process.env.RUBY_NPM_PACKAGE_ROOT) {
       expect(await resolve()).toBe("ok");
     });
   });
+
+  test.describe("JS::RequireRemote#load", () => {
+    test("JS::RequireRemote#load returns true", async ({ page }) => {
+      const resolve = await resolveBinding(page, "checkResolved");
+      await page.goto(
+        "https://cdn.jsdelivr.net/npm/ruby-head-wasm-wasi@latest/dist/",
+      );
+      await page.setContent(`
+      <script src="browser.script.iife.js"></script>
+      <script type="text/ruby" data-eval="async">
+        require 'js/require_remote'
+        JS.global.checkResolved JS::RequireRemote.instance.load 'error_on_load_twice'
+      </script>
+     `);
+
+      expect(await resolve()).toBe(true);
+    });
+
+    test("JS::RequireRemote#load returns false when same gem is loaded twice", async ({
+      page,
+    }) => {
+      const resolve = await resolveBinding(page, "checkResolved");
+      await page.goto(
+        "https://cdn.jsdelivr.net/npm/ruby-head-wasm-wasi@latest/dist/",
+      );
+      await page.setContent(`
+      <script src="browser.script.iife.js"></script>
+      <script type="text/ruby" data-eval="async">
+        require 'js/require_remote'
+        JS::RequireRemote.instance.load 'error_on_load_twice'
+        JS.global.checkResolved JS::RequireRemote.instance.load 'error_on_load_twice'
+      </script>
+     `);
+
+      expect(await resolve()).toBe(false);
+    });
+
+    test("JS::RequireRemote#load identifies by URL after redirect", async ({
```

Comment:
Under investigation

This test fails.
In the integration test, response#url returns the URL before the redirect.
I do not know the cause.